store/tikv/2pc.go

```diff
@@ -570,10 +580,19 @@ func (c *twoPhaseCommitter) execute(ctx goctx.Context) error {
 		c.mu.RUnlock()
 		if !committed && !undetermined {
 			twoPhaseCommitGP.Go(func() {
+				start := time.Now()
+				secondaryLockCleanupWorkerCounter.Inc()
+				defer func() {
+					secondaryLockCleanupWorkerCounter.Dec()
```

Comment:
Any need to use label values to distinguish commit and cleanup?

Reply:
Yes, I think it's better!